lib/services.js

```diff
@@ -62,6 +62,21 @@ function describeModels(app) {
       var ctor = method.restClass.ctor;
       if (!ctor || method.sharedMethod.isStatic) return;
       method.accepts = ctor.accepts.concat(method.accepts);
+
+      if(!method.accepts) return;
+      //Any extra http action arguments in the path need to be added to the
+      //angular resource actions as params
```

Comment:
Coding style:

 - please add an empty line after the return statement to visually delimit different functional blocks of code
 - add a space after trailing slash - `// Any`, `// angular`